From f37f1deed88a8096e416b05e8c2666de9d2bec81 Mon Sep 17 00:00:00 2001
From: Darren Etheridge <detheridge@ti.com>
Date: Wed, 4 Oct 2023 14:50:12 -0500
Subject: [PATCH 037/322] media: platform: img: vxd: force device node name

Force the encoder to always be /dev/video1 (assuming it is available).

This makes it easier to write simple test scripts because assuming that
video0 is decode and video1 is encode makes life much simpler.

Signed-off-by: Darren Etheridge <detheridge@ti.com>
Signed-off-by: Vignesh Raghavendra <vigneshr@ti.com>
---
 drivers/media/platform/img/vxe-vxd/encoder/vxe_v4l2.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/img/vxe-vxd/encoder/vxe_v4l2.c b/drivers/media/platform/img/vxe-vxd/encoder/vxe_v4l2.c
index 4ce97fdf2b5b..9c3608e85359 100644
--- a/drivers/media/platform/img/vxe-vxd/encoder/vxe_v4l2.c
+++ b/drivers/media/platform/img/vxe-vxd/encoder/vxe_v4l2.c
@@ -1846,7 +1846,7 @@ static int vxe_enc_probe(struct platform_device *pdev)
 		goto out_vid_dev;
 	}
 
-	ret = video_register_device(vfd, VFL_TYPE_VIDEO, 0);
+	ret = video_register_device(vfd, VFL_TYPE_VIDEO, 1);
 	if (ret) {
 		dev_err(&pdev->dev, "Failed to register video device\n");
 		goto out_vid_reg;
-- 
2.43.0

